cypress/e2e/common/cmsSteps.js

```diff
@@ -416,4 +416,102 @@ And('user remembers the source of selected promo image for the mini landing', ()
     cy.get('details img').then($el => {
         imageSrc = $el[0].getAttribute('src')
     })
+});
+
+Then('user selects {string} option from Operations dropdown for content with title {string}', (translateOption, title) => {
+    cy.get(`td:contains('${title}')`).siblings('td').find(`ul.dropbutton >li> a:contains("${translateOption}")`).click({ force: true });
+});
+
+Then('the page title is {string}', (title) => {
+    cy.get(`h2:contains('${title}')`).should('be.visible');
+});
+
+When('user clicks on {string} button to add translation', (addBtn) => {
+    cy.get(`li.add.dropbutton-action a:contains("${addBtn}")`).click({ force: true });
+});
+
+And('Remove button for image was translated as {string}', (removeBtn) => {
+    cy.get(`input[value='${removeBtn}']`).should('be.visible');
+});
+
+And('Related Resources section was translated as {string}', (relatedResources) => {
+    cy.get(`div#field-related-resources-add-more-wrapper h4:contains("${relatedResources}")`).should('be.visible');
+});
+
+And('Link section under related resources was translated as {string}', (linkText) => {
+    cy.get(`summary.seven-details__summary span:contains("${linkText}")`).should('be.visible');
+});
+
+And('dropdown to add link under related resources was translated to start with {string}', (dropDownText) => {
+    cy.get(`li.dropbutton__item.dropbutton__item--extrasmall.dropbutton-action input[value^="Añadir"]`).should('be.visible');
+});
+
+And('button to add citation was translated as {string}', (citationBtn) => {
+    cy.get(`input[value='${citationBtn}']`).should('be.visible');
+});
+
+And('{string} label is displayed', (labelListDesc) => {
+    cy.get(`div[class*="js-form-item-field-list-description-0-value"] label:contains("${labelListDesc}")`).should('be.visible');
+});
+
+And('body was translated as {string}', (bodyTranslation) => {
+    cy.get(`div[id*=field-article-body] h4:contains("${bodyTranslation}")`).should('be.visible');
+});
+
+And('heading was translated as {string}', (headingTranslation) => {
+    cy.get(`div.paragraphs-subform.js-form-wrapper.form-wrapper label:contains("${headingTranslation}")`).should('be.visible');
+});
+
+And('body content was translated as {string}', (contentHeadingTranslation) => {
+    cy.get(`div[id="edit-field-article-body-0-subform-field-body-section-content-wrapper"] label:contains("${contentHeadingTranslation}")`);
```

Comment:
definitely not a common step - locator is specific to article, move it there

Reply:
done